embeddings/get_started_with_embedding_tuning.ipynb

```diff
@@ -0,0 +1,1434 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #119.    def get_topk_documents(
</pre><p>change function name to <code>get_top_k_documents()</code> Most of our functions write it as 	<code>top_k</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>done</p>